### PR TITLE
tor: Update to 0.4.3.5

### PR DESCRIPTION
**Pull request by svschmel**
Tor v0.4.1.x reached its end of life on 2020-05-20.
https://trac.torproject.org/projects/tor/wiki/org/teams/NetworkTeam/CoreTorReleases

Update to version 0.4.3.5.

```
==============================================
Testsuite summary for tor 0.4.3.5
==============================================
# TOTAL: 24
# PASS:  21
# SKIP:  3
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
==============================================
```

This version works fine:
https://metrics.torproject.org/rs.html#details/AD0FCFD83EA4899DB34735E599701592DEA13690

### CODE REVIEW

File: components/network/tor/Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ BUILD_BITS=		64
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		tor
-COMPONENT_VERSION=	0.4.1.9
+COMPONENT_VERSION=	0.4.3.5
 COMPONENT_REVISION=	1
```
Comment (alarcher) [inline components/network/tor/Makefile]: You forgot to remove the COMPONENT_REVISION variable.

**Review by alarcher — CHANGES_REQUESTED** [review]
Excellent update thank you :)
Just a small favour, could you run 'gmake test'?
After that you need to copy 'build/test/amd64/results-64.snapshot' to 'test/results-64.master then commit the changes.
I'll merge after that :)

**Comment by svschmel** [comment]
Hello,

- COMPONENT_REVISION variable => removed 
- copy 'build/test/amd64/results-64.snapshot' to 'test/results-64.master => done
- commit => done

I hope this is it :)

Regards

**Comment by alarcher** [comment]
> Hello,
> 
>     * COMPONENT_REVISION variable => removed
> 
>     * copy 'build/test/amd64/results-64.snapshot' to 'test/results-64.master => done
> 
>     * commit => done
> 
> 
> I hope this is it :)
> 
> Regards

Great! thank you :)

**Review by alarcher — APPROVED** [review]
Squashed commits, all good!

**Pull request merged**